spyder_kernels/console/start.py

```diff
@@ -75,6 +80,56 @@ def sympy_config(mpl_backend):
     return lines
 
 
+def is_anaconda():
```

Comment:
@dalthviz, please move this function and the other functions you added below to `utils/anaconda.py` and import the necessary ones from there.